src/frontend/Sider/Behandling/VedtakOgBeregning/Læremidler/VedtakOgBeregningLæremidler.tsx

```diff
@@ -53,9 +62,10 @@ const VedtakOgBeregningLæremidler: FC = () => {
                         </HGrid>
                     </Panel>
 
-                    {typeVedtak === TypeVedtak.INNVILGELSE && (
-                        <span>Har ikke støtte for innvilgelse ennå</span>
-                        /*<InnvilgeBarnetilsyn lagretVedtak={vedtak as InnvilgelseBarnetilsyn} />*/
+                    {typeVedtak === TypeVedtak.INNVILGELSE && kanInnvilgeLæremidler ? (
+                        <InnvilgeLæremidler lagretVedtak={vedtak as InnvilgelseLæremidler} />
+                    ) : (
+                        <Alert variant="error">Støtte for innvilgelse er ikke laget enda</Alert>
```

Comment:
Kanskje ikke veldig viktig, men nå vil alert vises for avslag og :P

Reply:
Ai:(